api/admin/service.go

```diff
@@ -319,7 +346,7 @@ func (a *Admin) GetLoggerLevel(_ *http.Request, args *GetLoggerLevelArgs, reply
 }
 
 // GetConfig returns the config that the node was started with.
-func (a *Admin) GetConfig(_ *http.Request, _ *struct{}, reply *interface{}) error {
+func (a *Admin) GetConfig(_ *http.Request, args *Secret, reply *interface{}) error { //nolint:revive
```

Comment:
why not just instead of nolint?
```suggestion
func (a *Admin) GetConfig(_ *http.Request, _ *Secret, reply *interface{}) error { //nolint:revive
```

Reply:
args is use in reflect (ginkgo), you even cannot name it different.